.github/workflows/changesets.yml

```diff
@@ -81,6 +81,7 @@ jobs:
         run: node -r esbuild-register tools/deployments/alert-on-error.ts
         env:
           PUBLISH_STATUS: ${{ steps.changesets.outcome }}
+          HAS_CHANGESETS: ${{ steps.changesets.outputs.hasChangesets }}
```

Comment:
The `hasChangesets` output is mentioned in https://github.com/changesets/action?tab=readme-ov-file#custom-publishing